app/views/storage.html

```diff
@@ -30,11 +30,13 @@
             <div class="col-md-12">
               <div class="section-header page-header-bleed-right page-header-bleed-left">
                 <div class="hidden-xs pull-right" ng-if="project && ('persistentvolumeclaims' | canI : 'create')">
-                  <a ng-disabled="outOfClaims" ng-href="project/{{project.metadata.name}}/create-pvc" class="btn btn-default">Create Storage</a>
+                  <a ng-if="!outOfClaims" ng-href="project/{{project.metadata.name}}/create-pvc" class="btn btn-default">Create Storage</a>
+                  <a ng-if="outOfClaims" ng-href="" ng-disabled="true" class="btn btn-default">Create Storage</a>
```

Comment:
Just add the `disabled` class, and no need to use `ng-href`.

```html
<a ng-if="outOfClaims" href="" class="btn btn-default disabled">Create Storage</a>
```

Probably a good idea to tell screenreaders it's disabled, too, with `aria-disabled`


```html
<a ng-if="outOfClaims" href="" class="btn btn-default disabled" aria-disabled="true">Create Storage</a>
```

Also make the change to the same button below under `visible-xs-block`